Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AtomRenderPlugin.cpp

```diff
@@ -6,31 +6,52 @@
  *
  */
 
+#include <AzFramework/Viewport/ViewportControllerList.h>
+#include <AzFramework/Viewport/ViewportColors.h>
+#include <AzToolsFramework/Viewport/ViewportTypes.h>
+#include <AzToolsFramework/Viewport/ViewportSettings.h>
+#include <AzToolsFramework/Manipulators/ManipulatorManager.h>
+
 #include <EMStudio/AtomRenderPlugin.h>
 #include <EMStudio/AnimViewportRenderer.h>
 #include <EMStudio/AnimViewportToolBar.h>
+#include <EMStudio/AnimViewportInputController.h>
 
 #include <Integration/Components/ActorComponent.h>
 #include <EMotionFX/CommandSystem/Source/CommandManager.h>
 #include <EMotionFX/Source/Allocators.h>
 #include <EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/EMStudioManager.h>
 #include <QHBoxLayout>
+#include <QGuiApplication>
+
 
 namespace EMStudio
 {
     AZ_CLASS_ALLOCATOR_IMPL(AtomRenderPlugin, EMotionFX::EditorAllocator, 0);
+    const AzToolsFramework::ManipulatorManagerId g_animManipulatorManagerId =
+        AzToolsFramework::ManipulatorManagerId(AZ::Crc32("AnimManipulatorManagerId"));
 
     AtomRenderPlugin::AtomRenderPlugin()
         : DockWidgetPlugin()
+        , m_translationManipulators(
+              AzToolsFramework::TranslationManipulators::Dimensions::Three, AZ::Transform::Identity(), AZ::Vector3::CreateOne())
+        , m_rotateManipulators(AZ::Transform::CreateIdentity())
+        , m_scaleManipulators(AZ::Transform::CreateIdentity())
     {
     }
 
     AtomRenderPlugin::~AtomRenderPlugin()
     {
+        SaveRenderOptions();
         GetCommandManager()->RemoveCommandCallback(m_importActorCallback, false);
         GetCommandManager()->RemoveCommandCallback(m_removeActorCallback, false);
         delete m_importActorCallback;
         delete m_removeActorCallback;
+
+        if (ManipulatorRequestBus::Handler::BusIsConnected())
+        {
+            ManipulatorRequestBus::Handler::BusDisconnect();
+        }
```

Comment:
nit: This check happens internally so it's safe to just call `ManipulatorRequestBus::Handler::BusDisconnect();`